### DISCUSSION TITLE
Enqueue job by name/unique identifier

### DISCUSSION THREAD

**Discussion by king-cony** [jobrunr/jobrunr#742] [Q&A]
Hello,

I'm new to JobRunr and I'm really liking what I'm seeing! Is there anyway to enqueue a job using only the job's name (e.g. from the `@Job` annotation)? I'm trying to integrate JobRunr into a service for only queueing jobs, and I'd like the actual job code and processing to take place on a separate dedicated worker machine. This would be akin to Celery's `send_task` where the parameters are the module/package name of the task along with its arguments.

Thanks!

**Comment by rdehuyss** [comment]
Hi, I'm afraid this is only possible in the Pro version.

I should perhaps make it more explicit in the docs though.
